### PR TITLE
[Monit-28550] Configuration change to not forward resource attributes by default

### PR DESCRIPTION
**Pull request by deosu**

### CODE REVIEW

**Review by keep94 — COMMENTED** [review]
deosu: Great work on this PR, and good work on figuring out ProxyConfig.

File: proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpGrpcMetricsHandler.java [hunk 1]
```diff
@@ -20,40 +20,43 @@ public class OtlpGrpcMetricsHandler extends MetricsServiceGrpc.MetricsServiceImp
 
     private final ReportableEntityHandler<ReportPoint, String> pointHandler;
     private final ReportableEntityHandler<ReportPoint, String> histogramHandler;
-
     private final Supplier<ReportableEntityPreprocessor> preprocessorSupplier;
     private final String defaultSource;
+    private final Boolean includeResourceAttrs;
```
[the remark this answers is not in the document]
Comment (deosu) [reply proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpGrpcMetricsHandler.java:25]: I agree, no need to have a primitive object. changing the code.

File: proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufPointUtils.java [hunk 2]
```diff
@@ -99,6 +100,15 @@ private static List<ReportPoint> fromOtlpRequest(ExportMetricsServiceRequest req
     return wfPoints;
   }
 
+  @VisibleForTesting
+  static List<KeyValue> getResourceAttributes(Boolean includeResourceAttrs, Pair<String, List<KeyValue>> sourceAndResourceAttrs) {
+    List<KeyValue> resourceAttributes = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (deosu) [reply proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpProtobufPointUtils.java:105]: Agree, making changes.

**Review by keep94 — APPROVED** [review]

**Review by oppegard — APPROVED** [review]
LGTM – we paired to review and refactor this.

**Pull request merged**